### ISSUE TITLE
Column ordering for abagen.mouse fetchers is inconsistent

### ISSUE THREAD

**Issue by rmarkello** [rmarkello/abagen#56]
## The issue

This is horribly pedantic, but when you call `abagen.mouse.fetch_allenref_genes()` and `abagen.mouse.fetch_allenref_structures()` TWICE (the first time the data are grabbed from the web and cached, and the second time they are read from the cache) the columns of the resulting `pandas.DataFrame` are:

```python
['acronym', 'id', 'name']. 
```

However, when you call `abagen.mouse.fetch_rubinov2015_structures()` (any number of times) the columns are:

```python
['id', 'acronym', 'name']. 
```

It would be great for these to be in the same order!

## Proposed solution

For both `fetch_allenref_genes()` and `fetch_allenref_structures()` modifying the lines that read:

```
else:
    genes/structures = pd.read_csv(fname)
```

to:

```
else:
    genes/structures = pd.read_csv(fname)[entries]
```

would do the trick! The `entries` object is a list defined earlier in both functions that is ordered ['id', 'acronym', 'name'] and would ensure consistent outputs. This is how `fetch_rubinov2015_structures()` works so making them consistent would be great!

**Comment by abkosar** [comment]
Hey @rmarkello -- Are you looking for someone working on the issue?

**Comment by rmarkello** [comment]
Hi @abkosar ! Sorry for the delay&mdash;I was away from the internet for a while and am just getting back into things.

If you're willing to take a whack at this that would be awesome ! :raised_hands: If you'd be willing to add a test to ensure that the outputs are consistently the order specified, too, that would be great! Let me know if you have any questions as you go along.

**Comment by abkosar** [comment]
Hey @rmarkello! Sure sounds good to me! I will start working on it and create a PR when done! 

Thanks!

**Comment by rmarkello** [comment]
Closed by #62.
